policy/templates/el7-pgo-build/ci/goreleaser/goreleaser-el7.yml

```diff
@@ -8,10 +8,10 @@
 
   {{- template "before_hooks" . }}
 
-{{- if eq .Branch "master" }}
-  {{- template "builds" . }}
-{{- else }}
+{{- if or (eq .Branch "release-5-lts") (eq .Branch "release-4-lts") }}
```

Comment:
This conditional is going to be a problem. It is better to use `el7-pgo-build` for this conditional.

Reply:
Alok, I see your point. Adding the 'el7-pgo-build' for this conditional will change the current rendering results to something different than what we have today.

<img width="480" alt="image" src="https://github.com/TykTechnologies/gromit/assets/35462288/19a8cc02-68bd-45c5-a04d-9e97e60fd571">

'el7-pgo-build' Applies in cofig.yaml for all branches within tyk-analytics as you can see in the image above. That means el7 goreleaser file should be rendered only for release-5-lts and 4-lts, please confirm that's the behavior we want.
I will do the changes , if it's fine please approve and get it merge, otherwise lets sync on this tomorrow and wrap it up

Reply:
Done